tests/safeds/data/tabular/transformation/test_discretizer.py

```diff
@@ -0,0 +1,223 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.data.tabular.transformation import Discretizer
+from safeds.exceptions import TransformerNotFittedError, UnknownColumnNameError
+
+
+class TestInit:
+    def test_should_raise_value_error(self) -> None:
+        with pytest.raises(ValueError, match="Parameter 'number_of_bins' must be >= 2."):
+            _ = Discretizer(1)
+
+
+class TestFit:
+    def test_should_raise_if_column_not_found(self) -> None:
+        table = Table(
+            {
+                "col1": [0.0, 5.0, 10.0],
+            },
+        )
+
+        with pytest.raises(UnknownColumnNameError):
```

Comment:
Please test for the specific error message with the `match` parameter in `pytest.raises`